docs/source/articles/http-caching.html.md

```diff
@@ -31,7 +31,31 @@ This creates a major complication for any data on the page with user or session
 
 **Note:** [Rack::Cache](http://rtomayko.github.io/rack-cache/) is a simple way to achieve the caching advantage explained above without the configuration and hosting burden of a more robust HTTP cache like [Varnish](https://www.varnish-cache.org).
 
-## Customizing the Rack::Cache Key
+## Using Rack::Cache
+
+Many Workarea applications use `Rack::Cache` for storing their HTTP cache. In fact, this is the caching solution that is used by [Workarea Commerce Cloud](https://www.workarea.com/pages/commerce-cloud), so if you're on Commerce Cloud, congratulations! You're already using `Rack::Cache`! 🎉
+
+Now that we're done celebrating, we can focus on a few key elements of `Rack::Cache` that are configurable and notable for Workarea applications.
+
+### Ignore User-Specific Tracking Parameters in the Query String
+
+Many stores on the Workarea platform use marketing tools suck as [Listrak](https://github.com/workarea-commerce/workarea-listrak) or [Emarsys](https://github.com/workarea-commerce/workarea-emarsys) to handle abandoned cart emails, marketing campaigns, and so on. These services tend to use query parameters within the URL to identify each individual user, and this can cause an issue with caching. `Rack::Cache` will, by default, generate cache keys based on the given URL. So, if you have multiple requests to your **/categories/mens** page, like this:
```

Comment:
```suggestion
Many stores on the Workarea platform use marketing tools suck as [Listrak](https://github.com/workarea-commerce/workarea-listrak) or [Emarsys](https://github.com/workarea-commerce/workarea-emarsys) to handle abandoned cart emails, marketing campaigns, and so on. These services sometimes use query parameters within the URL to identify each individual user, and this can cause an issue with caching. `Rack::Cache` will, by default, generate cache keys based on the given URL. So, if you have multiple requests to your **/categories/shirts** page, like this:
```